source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -677,9 +687,86 @@ namespace aspect
 
         // create shortcuts
         const std::array<double, 4> &tau = ref_resolved_shear_stress;
-
         std::vector<double> strain_energy(n_grains);
         double mean_strain_energy = 0;
+        //debugging init
+        // if (deformation_type== DeformationType::olivine_d_fabric)
+        //   {
+        //     std::cout << "-------------------" <<std::endl;
+        //     std::cout << "\n def_type=olivined" <<std::endl;
+        //   }
+        // else if (deformation_type== DeformationType::clinopyroxene)
+        //   {
+        //     std::cout << "def_type=CPX" <<std::endl;
+        //   }
+
+        // for (int i = 3; i >= 0; i--)
+        //   std::cout << "outRRSS["<<i<<"]"<< ref_resolved_shear_stress[i]<<std::endl;
+        //debugging ends
```

Comment:
Remove all commented out code. Commented out code is confusing for readers, because it doesnt explain why it was commented, and it is certainly untested code, so it will break without anyone noticing.